README.md

```diff
@@ -34,10 +34,19 @@ generated file by hand.
 
 ## Usage
 
+In Unix command lines, use:
 ```
 node abp2blocklist.js < easylist.txt > easylist.json
 ```
 
+Alternately, the following command can be used in PowerShell, but it results in a larger filesize:
```

Comment:
Why would this result into a larger file size?

(CC: @mjethani)

Reply:
I can't tell for sure, but I could see that while the main command resulted in a ~80kB filesize, the PowerShell-friendly command resulted in a ~150kB filesize despite having the same text contents at first glance.